### PR TITLE
Added resolver Section

### PR DESCRIPTION
**Pull request by bhavik-m**

### CODE REVIEW

**Comment by KaymasJain** [comment]
Hey @bhavik-m! We need to add in different sections like this. So section for each resolver :) 
image

**Comment by pradyuman-verma** [comment]
It should be fixed
image

**Comment by pradyuman-verma** [comment]
MAINNET ->UNISWAP section not showing
image

**Comment by bhavik-m** [comment]
Can't figure out how to get rid of the "undefined" section
Screenshot (397)_LI

**Comment by bhavik-m** [comment]
Can't figure out How to get rid of 'undefined' section
Screenshot (397)_LI

File: content/en/get-started/cast (smart contract).md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Create a DSA Account using this function. It returns the address of the DSA acco
 |accountversion|uint256|Account Module version|
 |_origin|address|Where Smart Account is created|
 
-* [cast(_targets, _datas, _origin)](https://github.com/Instadapp/dsa-contracts/blob/master/contracts/v2/accounts/module1/Implementation_m1.sol#L80)
+* #### cast(_targets, _datas, _origin) [#] (https://github.com/Instadapp/dsa-contracts/blob/master/contracts/v2/accounts/module1/Implementation_m1.sol#L80)
```
Comment (pradyuman-verma) [inline content/en/get-started/cast (smart contract).md:86]:
```suggestion
* #### cast(_targets, _datas, _origin) [#](https://github.com/Instadapp/dsa-contracts/blob/master/contracts/v2/accounts/module1/Implementation_m1.sol#L80)
```

File: content/en/get-started/cast (smart contract).md [hunk 2]
```diff
@@ -74,7 +74,7 @@ This is the Main Contract for all the Defi Smart Accounts. Used to create a new
 * #### InstaImplementationM1 [#](https://github.com/Instadapp/dsa-contracts/blob/master/contracts/v2/accounts/module1/Implementation_m1.sol)
 This contract contains most core functions of smart account name cast(). It is only called by owners of smart accounts and has full-fledge access over the smart account. Used also to access all the DSA.
 
-* #### build(owner, accountVersion, _origin) [#](https://github.com/Instadapp/dsa-contracts/blob/master/contracts/registry/index.sol#L170)
+* #### build(owner, accountVersion, _origin)[#](https://github.com/Instadapp/dsa-contracts/blob/master/contracts/registry/index.sol#L170)
```
Comment (pradyuman-verma) [inline content/en/get-started/cast (smart contract).md:77]:
```suggestion
* #### build(owner, accountVersion, _origin) [#](https://github.com/Instadapp/dsa-contracts/blob/master/contracts/registry/index.sol#L170)
```

File: content/en/resolvers/mainnet.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+description: "Avalanche resolvers"
+position: 10
+category: "resolvers"
+---
+
+* [Compound](./mainnet/compound)
+
+* [Aave](./mainnet/aave)
+
+* [MakerDAO](./mainnet/MakerDAO)
```
Comment (pradyuman-verma) [inline content/en/resolvers/mainnet.md:13]:
```suggestion
* [MakerDAO](./mainnet/makerdao)
```

**Pull request merged**